tests/kafkatest/tests/streams/streams_broker_bounce_test.py

```diff
@@ -133,6 +134,15 @@ def fail_many_brokers(self, failure_mode, num_failures):
         for num in range(0, num_failures - 1):
             signal_node(self, self.kafka.nodes[num], sig)
 
+    def get_topics_count(self):
+        count = 0
```

Comment:
It seems we are counting topics multiple times. Can we use a set instead of count, and add the topics from the `topic_list` to the set and return the size of the set instead?